react/features/whiteboard/constants.ts

```diff
@@ -52,3 +52,8 @@ export const WHITEBOARD_UI_OPTIONS = {
         }
     }
 };
+
+/**
+ * Whiteboard soft limit factor.
+ */
+export const USER_LIMIT_THRESHOLD = 0.25;
```

Comment:
Not sure a percentage is a good idea here. I'd say we set it to 5 and that's that. If you set it to 100 we'd be acommodating too many more for example.

Reply:
What if the limit is below 5?

Reply:
That makes no sense, we should not support anything less than 10.

Reply:
Yes, let's limit the lower boud to 10.